articles/api/overview.md

```diff
@@ -0,0 +1,83 @@
+---
+title: APIs Overview
+description: This page explains the basics of APIs, their role in OAuth and how to configure an API in Auth0 Dashboard.
+section: apis
+url: /apis
+---
+
+# APIs
+
+## Overview
+
+An API is an entity that represents an external resource, capable of accepting and responding to protected resource requests made by clients. At the [OAuth2 spec](https://tools.ietf.org/html/rfc6749) an API maps to the **Resource Server**.
+
+When a client wants to access an API's protected resources it must provide an access token. The same access token can be used to access the API's resources without having to authenticate again, until it expires.
+
+Each access token may include a list of the permissions that have been granted to the client. These permissions are defined using the **scope** request parameter.
+
+For example, an API that holds a user's appointments, may accept two different levels of authorization: read only (scope `read:appointments`) or write (scope `write:appointments`). When a client asks the API to list a user's appointments, then the access token should contain the `read:appointments` scope. In order to edit an existing appointment or create a new one, the access token should contain the `write:appointments` scope.
+
+> For more information on tokens please refer to: [Tokens used by Auth0](/tokens).
+
+## How to configure an API in Auth0
```

Comment:
Just FYI, this subsection will be removed at some point in the future when we make the section available by default.